go/engine/login_provision.go

```diff
@@ -502,6 +502,7 @@ func (e *loginProvision) deviceName(ctx *Context) (string, error) {
 			return "", err
 		}
 		if !libkb.CheckDeviceName.F(devname) {
+			e.G().Log.Debug("invalid device name supplied: %s", devname)
```

Comment:
👍

Reply:
needed that, the logs were quite mute on this topic.

Reply:
Yeah totally empty, this made the bug obvious.